lab_6_pipeline/pipeline.py

```diff
@@ -189,6 +295,14 @@ def _count_frequencies(self, article: Article) -> dict[str, int]:
         Returns:
             dict[str, int]: POS frequencies
         """
+        frequencies = {}
+        for conllu_sentence in self._analyzer.from_conllu(article).sentences:
+            for word in conllu_sentence.words:
+                word_feature = word.to_dict()['upos']
+                if word_feature not in frequencies:
+                    frequencies[word_feature] = frequencies.get(word_feature, 0)
```

Comment:
Why use `get` if you've established that the item is not the dictionary? 
You may skip the condition altogether.

Reply:
Tried to change it